app.js

```diff
@@ -0,0 +1,10 @@
+const express = require ('express');
+const app= express();
+const fs = require ('fs/promises');
+const {getApi, getApiTopics} = require('./controllers/api.controller');
+
+app.use(express.json());
```

Comment:
You don't need this until you make a POST request, its not a good idea to include it yet.